Resources/Locale/en-US/_DV/clothing/juggernaut.ftl

```diff
@@ -0,0 +1,5 @@
+#Juggernaut suit helmet is no longer a dome
+ent-ClothingHeadHelmetHardsuitCybersun = cybersun juggernaut helmet
+    .desc = A helmet designed in the Tau chromosphere facility capable of shrugging off withering volumes of fire.
+ent-ClothingHeadHelmetHardsuitJuggernautReverseEngineered = SA-127 combat hardsuit helmet
```

Comment:
REd one is from deltav so just edit it, its not from upstream

also this file is missing a trailing newline